mephisto/abstractions/database.py

```diff
@@ -983,6 +984,23 @@ def delete_qualification(self, qualification_name: str) -> None:
             provider = ProviderClass(self)
             provider.cleanup_qualification(qualification_name)
 
+    @abstractmethod
+    def _find_granted_qualifications(
+        self, worker_id: Optional[str] = None,
+    ) -> List[GrantedQualification]:
+        """find_granted_qualifications implementation"""
+        raise NotImplementedError()
+
+    @FIND_GRANT_QUALIFICATION_LATENCY.time()
+    def find_granted_qualifications(
+        self, worker_id: Optional[str] = None,
+    ) -> List[GrantedQualification]:
+        """
+        Find granted qualifications.
+        If `worker_id` is not supplied, returns all granted qualifications.
+        """
+        return self._find_granted_qualifications(worker_id=worker_id)
```

Comment:
This is definitely a nomenclature issue from me from way back. We should be using the signature of `check_granted_qualifications` over a `worker_id` only setup. I don't really know why I made the 'find' for `granted_qualifications` "`check`". Can you have `check_granted_qualifications` point to this as well?

Reply:
We'll have to go through deprecation steps. Make `find_granted_qualifications` a direct duplicate of `check_granted_qualifications`, and make the latter point to the former. We can then add a pending deprecation warning.